tracer/src/Datadog.Trace/IAST/Range.cs

```diff
@@ -9,7 +9,7 @@ namespace Datadog.Trace.Iast;
 
 internal readonly struct Range
 {
-    public Range(int start, int length, Source? source)
+    public Range(int start, int length, Source source)
```

Comment:
Is Source _actually_ always required now? Or is this change just because it moved from struct -> class?

Reply:
It is always required now. Thinking about it, it makes no sense a range created from a null source.